### PR TITLE
[libvpx] Add cmake config file

### PR DESCRIPTION
**Pull request by mcmtroffaes**
This allows libvpx to be used with

find_package(libvpx CONFIG REQUIRED)
target_link_libraries(main PRIVATE libvpx::libvpx)

**Describe the pull request**

- What does your PR fix? See above.

- Which triplets are supported/not supported? Have you updated the CI baseline? Tested on all supported windows triplets. Should work also on linux/darwin provided the same library naming convention is used (vpx base name for release, vpxd base name for debug). If not, it is easy to add alternate names.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? Yes.

### CODE REVIEW

**Comment by traversaro** [comment]
Unofficial targets created by vcpkg ports are typically prefixed by `unofficial` (see https://github.com/microsoft/vcpkg/blob/a4a8b8ebdd1b9d698c4ec92221fbca722e6e3e0e/ports/libiconv/unofficial-iconv-config.cmake#L1), could it make sense to do the same here?

**Comment by mcmtroffaes** [comment]
That is a splendid suggestion @traversaro - I'll go ahead and make that change.

**Comment by mcmtroffaes** [comment]
I've done the change suggested by @traversaro, and I've also added a target guard in case the config gets loaded more than once (similar to what iconv is doing), since that seemed good practice. With those changes, the library can be used as follows:

```cmake
find_package(unofficial-libvpx CONFIG REQUIRED)
target_link_libraries(main PRIVATE unofficial::libvpx::libvpx)
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please update the version info. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

File: ports/libvpx/unofficial-libvpx-config.cmake.in [hunk 1]
```diff
@@ -0,0 +1,49 @@
+if(NOT TARGET unofficial::libvpx::libvpx)
+  # Compute the installation prefix relative to this file.
+  get_filename_component(_IMPORT_PREFIX "${CMAKE_CURRENT_LIST_FILE}" PATH)
+  get_filename_component(_IMPORT_PREFIX "${_IMPORT_PREFIX}" PATH)
+  get_filename_component(_IMPORT_PREFIX "${_IMPORT_PREFIX}" PATH)
+
+  # Add library target (note: vpx always has a static build in vcpkg).
+  add_library(unofficial::libvpx::libvpx STATIC IMPORTED)
```
Comment (JackBoosY) [inline ports/libvpx/unofficial-libvpx-config.cmake.in:8]: Should this always be static?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by dan-shaw** [comment]
Thanks for the PR!

**Pull request merged**